README.md

```diff
@@ -5,18 +5,21 @@ Structured Data Prediction Error Estimation
 ## Install
 
 Project dependencies are in `pyproject.toml` file.
-
+We recommend installing `Mambaforge`, which is a conda installation with `mamba` installed by default and set to use `conda-forge` as the default channel.
```

Comment:
oh do i need to get mamba now? any tradeoffs with conda, or is it just better all around?

Reply:
Better all around. It's super fast and it does the exact same thing as conda.